packages/editor/src/client/panel/imageSelectPanel.tsx

```diff
@@ -67,7 +67,7 @@ export function ImageSelectPanel({onClose, onSelect}: ImageSelectPanelProps) {
 
   function loadMore() {
     fetchMore({
-      variables: {first: ImagesPerPage, after: data?.images.pageInfo.endCursor},
+      variables: {take: ImagesPerPage, skip: 1, cursor: data?.images.pageInfo.endCursor},
```

Comment:
cursor based navigation needs a skip 1 else it will return the cursor

Reply:
When I want to query the first page of some data (e.g. articles), do I also have to skip: 1? That would be an anti-pattern imo.